component/authenticator/src/main/java/org/wso2/carbon/identity/authenticator/emailotp/EmailOTPAuthenticator.java

```diff
@@ -260,7 +263,16 @@ private void checkEmailOTPBehaviour(AuthenticationContext context, Map<String, S
                                         Map<String, String> authenticatorProperties, String email, String username,
                                         String myToken) throws AuthenticationFailedException {
         if (isSMTP(authenticatorProperties, emailOTPParameters, context)) {
-            sendOTP(username, myToken, email);
+            // Check whether the authenticator is configured to use the event handler implementation.
+            if (emailOTPParameters.get(EmailOTPAuthenticatorConstants.USE_EVENT_HANDLER_BASED_EMAIL_SENDER) != null &&
+                    Boolean.parseBoolean(emailOTPParameters.get(EmailOTPAuthenticatorConstants.USE_EVENT_HANDLER_BASED_EMAIL_SENDER))) {
```

Comment:
Format the code to wso2 standards code styles

Reply:
Fixed https://github.com/wso2-extensions/identity-outbound-auth-email-otp/pull/26/commits/4c8a1a700c541009035ba290999b9609af6e95d2